lib/server.js

```diff
@@ -0,0 +1,26 @@
+var express = require('express'),
+    bodyParser = require('body-parser'),
+    buildResponses = require('./buildResponses'),
+    config = require('../config');
+
+var app = express();
+
+app.use(bodyParser.json());
+
+app.post('/sms/v2/smsoutbound', function(request, response){
```

Comment:
Try to avoid anonymous functions.

Better define first function smsHandler() {}, and app.post('/sms/v2/smsoutbound', smsHandler) later